packages/components/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,8 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+type TagProps = {
+    children: React.ReactNode;
+};
+export const Tag = ({ children }: TagProps) => {
```

Comment:
```suggestion
};

export const Tag = ({ children }: TagProps) => {
```
Let's bring some air 🌬️